test/functional/transactions/0.tx.js

```diff
@@ -0,0 +1,1001 @@
+					node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Failed to validate transaction schema: Expected type integer but found type number');
+					badTransactions.push(transaction);
+					done();
+				});
+			});
+
+			it('using more than maximum should fail', function (done) {
+				transaction.amount = Number(new BigNumber(constants.totalAmount) + 1);
+
+				sendTransaction(transaction, function (err, res) {
+					node.expect(res).to.have.property('success').to.be.not.ok;
+					node.expect(res).to.have.property('message').to.have.string('Invalid transaction body - Failed to validate transaction schema: Value '+transaction.amount+' is greater than maximum 10000000000000000');
+					badTransactions.push(transaction);
+					done();
+				});
+			});
+		});
+
+		describe('sending transactions with wrong FEE values', function () {
+
+			it('using null should fail', function (done) {
+				transaction.fee = null;
+
+				sendTransaction(transaction, function (err, res) {
+					node.expect(res).to.have.property('success').to.not.be.ok;
+					node.expect(res).to.have.property('message').to.equal('Invalid transaction fee');
+					badTransactions.push(transaction);
+					done();
+				}, true);
+			});
+
+			it('using undefined should fail', function (done) {
+				transaction.fee = undefined;
+
+				sendTransaction(transaction, function (err, res) {
+					node.expect(res).to.have.property('success').to.not.be.ok;
+					node.expect(res).to.have.property('message').to.equal('Invalid transaction fee');
+					badTransactions.push(transaction);
+					done();
+				}, true);
+			});
+
+			it('using NaN should fail', function (done) {
+				transaction.fee = NaN;
+
+				sendTransaction(transaction, function (err, res) {
+					node.expect(res).to.have.property('success').to.not.be.ok;
+					node.expect(res).to.have.property('message').to.equal('Invalid transaction fee');
+					badTransactions.push(transaction);
+					done();
+				}, true);
+			});
+
+			it('using string should fail', function (done) {
+				transaction.fee = '';
+
+				sendTransaction(transaction, function (err, res) {
+					node.expect(res).to.have.property('success').to.not.be.ok;
+					// TODO
+					node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Failed to validate transaction schema: Expected type integer but found type string');
+					badTransactions.push(transaction);
+					done();
+				}, true);
+			});
+
+			it('using array should fail', function (done) {
+				transaction.fee = [];
+
+				sendTransaction(transaction, function (err, res) {
+					node.expect(res).to.have.property('success').to.not.be.ok;
+					// TODO
+					node.expect(res).to.have.property('message').to.equal('Invalid transaction body - Failed to validate transaction schema: Expected type integer but found type array');
+					badTransactions.push(transaction);
+					done();
+				}, true);
+			});
+
+			it('using object should fail', function (done) {
```

Comment:
consistency - before same test had different description, imo this one better, maybe even `empty object`